mx.jruby/suite.py

```diff
@@ -0,0 +1,83 @@
+suite = {
+  "mxversion" : "5.6.6",
+  "name" : "jrubytruffle",
+  "defaultLicense" : "GPLv2-CPE", # FIXME
+
+  "imports" : {
+    "suites": [
+            {
+                "name": "truffle",
+                "version": "551e8475af2fc8769bc3ead07c9156fe0ccbe338",
+                "urls": [
+                    {"url": "ssh://git@ol-bitbucket.us.oracle.com:7999/g/truffle.git",
+                     "kind": "git"},
+                    { "url" : "https://curio.ssw.jku.at/nexus/content/repositories/snapshots", "kind" : "binary" },
+                ]
+            },
+    ],
+  },
+
+  "projects" : {
+
+    # ------------- Projects -------------
+
+    # truffle/target/generated-sources/org/jruby/truffle/runtime/layouts
+    # NOT WORKING since mutual dependencies :/
+    # "org/jruby/truffle/runtime/layouts" : {
+    #   "subDir": "truffle/target/generated-sources",
+    #   "sourceDirs": [""],
+    #   "dependencies": [
+    #     "JRUBY_CORE",
+    #     "truffle:TRUFFLE_API",
+    #   ],
+    #   "javaCompliance" : "1.7",
+    # },
+
+    # truffle/target/generated-sources/antlr4
+    "generated-sources/antlr4" : {
+      "subDir": "truffle/target",
+      "sourceDirs": ["."],
+      "dependencies": [
+      ],
+      "javaCompliance" : "1.7",
+    },
+
+    # truffle/src/main/java
+    "truffle/src/main/java" : {
+      "subDir" : "",
+      "sourceDirs" : ["."],
```

Comment:
Maybe I could completely remove the projects and keep just the distribution? The projects are only useful for complication and for IDE support. But compilation is done by Maven and we don't need anything special to support IDEs - they already understand Maven structure.